.gitlab/source_test/linux.yml

```diff
@@ -7,7 +7,6 @@
   rm -f vendor.tar.gz
 
 .linux_tests_preparation:
```

Comment:
We could add something like:
```yaml
script:
  - inv -e test --race --profile --python-runtimes "$PYTHON_RUNTIMES" --cpus 4 $EXTRA_OPTS
```

to `.linux_tests_preparation` and set `$EXTRA_OPTS` to `--build-exclude=systemd`when necessary; that way we avoid some repetition.